### PR TITLE
Add `user.update` EventSub

### PR DESCRIPTION
**Pull request by BarryCarlyon**
> twitch event trigger user.update -F "URL" -s "YOURSECRET"

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

## Problem/Feature

One of my projects was erroring on a processing of `user.update` topic.
So I went to test it and found that the cli doesn't support it
This adds it

## Description of Changes: 

- Add a model in folder `user` of `user.update`
- Add a test for that model that might be wrong, not sure how to run tests
- Added UserId to the `EventsubCondition` model
- Enabled the mode in `type/stypes`

## Checklist

- [X] My code follows the [Contribution Guide](https://github.com/twitchdev/twitch-cli/blob/master/CONTRIBUTING.md#Requirements)
- [X] I have self-reviewed the changes being requested
- [X] I have made comments on pieces of code that may be difficult to understand for other editors
- [x] I have updated the documentation (if applicable)

### CODE REVIEW

File: internal/events/types/user/user_update_test.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "github.com/twitchdev/twitch-cli/test_setup"
+)
+
+var toUser = "4567"
+
+func TestEventSub(t *testing.T) {
+    a := test_setup.SetupTestEnv(t)
+
+    params := *&events.MockEventParameters{
+        UserID:     toUser,
```
Comment (lleadbet) [inline internal/events/types/user/user_update_test.go:20]: Do you mean either the FromUser or ToUser? That's why the tests are failing atm.

**Review by lleadbet — CHANGES_REQUESTED** [review]
Just need changes to the tests, otherwise lgtm.

File: internal/events/types/user/user_update_test.go (hunk 1 above)
Comment (BarryCarlyon) [reply internal/events/types/user/user_update_test.go:20]: I used ToUser in the lib and the test here

see
https://github.com/BarryCarlyon/twitch-cli/blob/user-update/internal/events/types/user/user_update.go#L40
https://github.com/BarryCarlyon/twitch-cli/blob/user-update/internal/events/types/user/user_update.go#L50

The test useds line 14 in here to spool up.

And my layout of the lib and test matches other types in here.

So I'm none the wiser here

Comment (BarryCarlyon) [reply internal/events/types/user/user_update_test.go:20]: oh I think I understand

Comment (BarryCarlyon) [reply internal/events/types/user/user_update_test.go:20]: nope it's tripped again :-|

**Review by lleadbet — APPROVED** [review]

**Pull request merged**